aten/src/ATen/miopen/Descriptors.cpp

```diff
@@ -22,7 +22,7 @@ void TensorDescriptor::set(const at::Tensor &t, size_t pad) {
   set(getDataType(t), t.sizes(), t.strides(), pad);
 }
 
-static int MIOPEN_DIM_MAX = 4;
+static int MIOPEN_DIM_MAX = 5;
```

Comment:
Is this an update in new MIOpen version?

Reply:
This was added to support the RNN ops with MIOpen. Not all MIOpen ops are supported with 5d tensors.